rootconf/grub-arch/sysroot-lib-onie/onie-blkdev-common

```diff
@@ -289,7 +289,7 @@ uefi_rm_stale_bootnum()
 {
     efibootmgr -v | grep ')/File(' | awk '{ print $1 }' | while read boot_num ; do
         local boot_num=${boot_num%\*}
-        local boot_path=$(efibootmgr -v | grep $boot_num | sed -e 's/.*)\/File(//' -e 's/)//' -e 's/\\/\//g')
+        local boot_path=$(efibootmgr -v | grep $boot_num | sed -e 's/.*)\/File(//' -e 's/).*//')
```

Comment:
I think zapping everything after the ')' is very reasonable.

However, this patch dropped the last sed statement `-e 's/\\/\//g'`, which provides a valuable service on some systems I have seen.  In the file path, this statement translates MS-DOS style back-slashes `\` into UNIX style forward-slashes `/`.  I think we need to keep that part.

Reply:
Sure - let me test it out with that part back in (I'd misunderstood what it did) and confirm it works as expected.

Reply:
I put it back in and everything is working great. I think the change shows up below - let me know if it doesn't :)
Mark